app/pages/solutions-list/filterType/covid19/ui.test.js

```diff
@@ -31,75 +21,65 @@ test('should display the page title', async (t) => {
   const pageTitle = Selector('h1[data-test-id="general-page-title"]');
   await t
     .expect(pageTitle.exists).ok()
-    .expect(await extractInnerText(pageTitle)).eql('Foundation Solution Sets – results');
+    .expect(await extractInnerText(pageTitle)).eql('Catalogue Solutions to help with coronavirus');
 });
 
 test('should display the page description', async (t) => {
   await pageSetup(t);
   const pageDescription = Selector('h2[data-test-id="general-page-description"]');
   await t
     .expect(pageDescription.exists).ok()
-    .expect(await extractInnerText(pageDescription)).eql('These Catalogue Solutions meet the 6 Foundation Capabilities that are the minimum requirement to enable a GP practice to operate.');
+    .expect(await extractInnerText(pageDescription)).eql('The following Catalogue Solutions can help prevent the spread of coronavirus by providing online consultations, services and information.');
 });
 
-test('should display the capabilities heading', async (t) => {
+test('should display the covid19 solution cards', async (t) => {
   await pageSetup(t);
-  const capabilityHeading = Selector('div[data-test-id="capability-list"] h5');
   await t
-    .expect(capabilityHeading.exists).ok()
-    .expect(await extractInnerText(capabilityHeading)).eql('Capabilities met');
+    .expect(Selector('div[data-test-id="solution-card-covid19"]').count).eql(4);
 });
 
-test('should display the foundation solution cards', async (t) => {
+test('should display the covid19 solution details of a solution card', async (t) => {
   await pageSetup(t);
+  const solutionCardsSection = Selector('div[data-test-id="solution-cards-covid19"]');
   await t
-    .expect(Selector('div[data-test-id="solution-card"]').count).eql(2);
-});
+    .expect(solutionCardsSection.find('div[data-test-id="solution-card-covid19"]').count).eql(4);
 
-test('should display the foundation solution details of a solution card', async (t) => {
-  await pageSetup(t);
-  const solutionCardsSection = Selector('div[data-test-id="solution-cards"]');
+  const solutionCard = solutionCardsSection.find('div[data-test-id="solution-card-covid19"]:nth-child(1)');
+  const covid19Tag = solutionCard.find('div[data-test-id="solution-card-covid19-tag"]');
   await t
-    .expect(solutionCardsSection.find('div[data-test-id="solution-card"]').count).eql(2);
+    .expect(covid19Tag.exists).ok()
+    .expect(await extractInnerText(covid19Tag)).eql('Coronavirus')
+    .expect(await extractInnerText(solutionCard.find('h4[data-test-id="solution-card-supplier"]'))).eql('iPLATO Healthcare Limited')
```

Comment:
can we get these strings straight from the data instead of hard coding them as it will make it easier to maintain

Reply:
this is how were doing it for foundation

Reply:
you can amend that file too if you like 😄

Reply:
these solutions might be changing quite a bit so better to have it dynamic

Reply:
I actually think we use this method in loads of places, should we not create a tech debt ticket?

Reply:
how about you start doing it here and we create a tech debt ticket for the rest? If we have identified it we should start doing it rather than making more tech debt